.github/workflows/build-brs.yml

```diff
@@ -0,0 +1,50 @@
+name: Build BRS
```

Comment:
Remove file build-brs.yml,  Tao Fang is working on another PR to figure out how to build and test brs on CI